modules/ingest-common/build.gradle

```diff
@@ -25,6 +25,7 @@ esplugin {
 
 dependencies {
   compileOnly project(':modules:lang-painless')
+  compile project(':libs:elasticsearch-core')
```

Comment:
Why is this necessary? modules depend on server, implicitly, so we should be able to use the date classes moved there from server, not need them in core lib

Reply:
I had problem accessing `IsoCalendarDataProvider` from server when running ingest yml tests.
Maybe adding this as test only? But then again it has to be available when running a cluster for yml test.
Any ideas?

Reply:
interesting. I cannot reproduce that failure now.  I can see from dependency tree that server is indeed on a test compile classpath. Not sure how was this failing for me before..
Reverted back the change moving `IsoCalendarDataProvider`